src/middlewared/middlewared/plugins/system/debug.py

```diff
@@ -131,9 +132,9 @@ def debug(self, job):
                     my_hostname = network['hostname_b']
                     remote_hostname = network['hostname']
 
-                tar.add(debug_job.result, f'{my_hostname}.txz')
+                tar.add(debug_job.result, f'{my_hostname}_{"active" if host_status == "MASTER" else "standby"}.txz')
```

Comment:
failover.status can return many different statuses. Including ERROR. We need to play it safe and be more explicit with the check. i.e. something like
```
if failover_status == 'MASTER':
    _active
else:
    don't_rename_archive